core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java

```diff
@@ -106,102 +111,56 @@ public void generateResponse(StaplerRequest2 req, StaplerResponse2 rsp, Object o
         }
 
         public ContextMenu add(String url, String text) {
```

Comment:
This file needs validating (maybe a clean up if possible without breaking plugins...)